### PR TITLE
Add municipality and fare zone seed data

### PR DESCRIPTION
**Pull request by culka**
This change is [Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807)

### CODE REVIEW

**Review by HenrikHartiala — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r1, 1 of 1 files at r2, 3 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807)* status: all files reviewed, 3 unresolved discussions (waiting on @culka)

___
*[`-- commits` line 5 at r2](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-Nve_3zy3Z9pFtyjTImg):*
Tiamat is rather "new" addition to our environment. But this commit is also changing the port of timetablesapi. Maybe some description would be nice in this commit body explaining why this change.
___
*[`scripts/seed-municipalities-and-fare-zones.sh` line 8 at r3](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-Nve__TE147MM-UDx2it:-Nve__TE147MM-UDx2iu:bo0cem2) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/7a8621be9561f0b27eeab7869c4afbe2bdbb2023/scripts/seed-municipalities-and-fare-zones.sh#L8)):*
> ```Shell
> echo "Uploading municipality and fare zones to localhost:$1"
> 
> curl --silent --output /dev/null --show-error --fail -X POST -H"Content-Type: application/xml" -d @netex/hsl-zones-netex.xml localhost:$1/services/stop_places/netex
> ```

no newline
___
*[`scripts/start-dependencies.sh` line 92 at r3](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-Nve_gTL26Cj6dwPiBN-:-Nve_gTL26Cj6dwPiBN0:bo0cem2) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/7a8621be9561f0b27eeab7869c4afbe2bdbb2023/scripts/start-dependencies.sh#L92)):*
> ```Shell
>   ./scripts/seed-municipalities-and-fare-zones.sh 3110 &
> fi
> wait
> ```

no newline

File: scripts/start-dependencies.sh [hunk 1]
```diff
@@ -85,3 +85,8 @@ if [ "$INCLUDE_E2E" = true ]; then
   ./scripts/seed-infrastructure-links.sh testdb-e2e &
 fi
 wait
+./scripts/seed-municipalities-and-fare-zones.sh 3010 &
```
Comment (jarkkoka) [inline scripts/start-dependencies.sh:88]: Please add a comment into which server this seed data is intended to be loaded into. The magic port number `3010` itse does not tell much to the reader.

File: scripts/start-dependencies.sh [hunk 2]
```diff
@@ -85,3 +85,8 @@ if [ "$INCLUDE_E2E" = true ]; then
...
+if [ "$INCLUDE_E2E" = true ]; then
+  ./scripts/seed-municipalities-and-fare-zones.sh 3110 &
```
Comment (jarkkoka) [inline scripts/start-dependencies.sh:90]: Please add a comment into which server this seed data is intended to be loaded into. The magic port number `3110` itse does not tell much to the reader.

**Review by jarkkoka — CHANGES_REQUESTED** [review]

**Review by culka — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807)* status: 1 of 5 files reviewed, 5 unresolved discussions (waiting on @HenrikHartiala and @jarkkoka)

___
*[`-- commits` line 5 at r2](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-Nve_3zy3Z9pFtyjTImg):*
<details><summary><i>Previously, HenrikHartiala wrote…</i></summary><blockquote>

Tiamat is rather "new" addition to our environment. But this commit is also changing the port of timetablesapi. Maybe some description would be nice in this commit body explaining why this change.
</blockquote></details>

Amended the comment for the commit with this info
___
*[`scripts/seed-municipalities-and-fare-zones.sh` line 8 at r3](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-Nve__TE147MM-UDx2it:-NveysJQ4wv753cMiaGV:bmkc4b3) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/7a8621be9561f0b27eeab7869c4afbe2bdbb2023/scripts/seed-municipalities-and-fare-zones.sh#L8)):*
<details><summary><i>Previously, HenrikHartiala wrote…</i></summary><blockquote>

no newline
</blockquote></details>

Newline
___
*[`scripts/start-dependencies.sh` line 88 at r3](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#gh-1568279457:-NveyurQEpV9c3IdwWFC:b7s3fui) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/7a8621be9561f0b27eeab7869c4afbe2bdbb2023/scripts/start-dependencies.sh#L88)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Please add a comment into which server this seed data is intended to be loaded into. The magic port number `3010` itse does not tell much to the reader.
</blockquote></details>

Comment added
___
*[`scripts/start-dependencies.sh` line 90 at r3](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#gh-1568279497:-Nveyvo3C-OO4gpIJ3jx:b7s3fui) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/7a8621be9561f0b27eeab7869c4afbe2bdbb2023/scripts/start-dependencies.sh#L90)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Please add a comment into which server this seed data is intended to be loaded into. The magic port number `3110` itse does not tell much to the reader.
</blockquote></details>

Comment added
___
*[`scripts/start-dependencies.sh` line 92 at r3](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-Nve_gTL26Cj6dwPiBN-:-NveytxMCX9_akaCyL9-:bmkc4b3) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/7a8621be9561f0b27eeab7869c4afbe2bdbb2023/scripts/start-dependencies.sh#L92)):*
<details><summary><i>Previously, HenrikHartiala wrote…</i></summary><blockquote>

no newline
</blockquote></details>

Newline

**Review by HenrikHartiala — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807)* status: 1 of 5 files reviewed, 2 unresolved discussions (waiting on @jarkkoka)

___
*[`-- commits` line 5 at r2](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-Nve_3zy3Z9pFtyjTImg):*
<details><summary><i>Previously, culka (Teemu Mäkinen) wrote…</i></summary><blockquote>

Amended the comment for the commit with this info
</blockquote></details>

Thanks, thats better. Although I still don't understand why the ports worked before this, or why hasn't things collapsed :D But maybe I just lack some other information. Descriptions are instantly better now anyway :+1:

**Review by jarkkoka — APPROVED** [review]
GML axis order is a complicated thing, but after googling the order used here (latitude/longitude) seems to be the default in GML 3.2.

> Reviewed 4 of 4 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/807#-:-NvfNDfh-W5U2N1qixRR:b-hbpnj3)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @culka)

**Pull request merged**